go-controller/pkg/util/ovs.go

```diff
@@ -432,6 +432,38 @@ func RunOVNNbctl(args ...string) (string, string, error) {
 	return RunOVNNbctlWithTimeout(ovsCommandTimeout, args...)
 }
 
+// NBTxn hold parts of an ovn-nbctl transaction request
+type NBTxn struct {
+	args    []string
+	txnArgs []string
+	env     []string
+}
+
+// NewNBTxn returns a new ovn-nbctl transaction request object
+func NewNBTxn() *NBTxn {
+	args, env := getNbctlArgsAndEnv(ovsCommandTimeout, []string{}...)
```

Comment:
is guaranteed that it always return the right `args` and `env` parameters?
or should it fail to create the new object?

Reply:
Yeah, that function has no way to return an error so I think we are OK.